ui/nuxeo-data-table/iron-data-table.js

```diff
@@ -214,6 +214,13 @@ import '../nuxeo-button-styles.js';
             display: block;
             @apply --nuxeo-label;
           }
+
+          /* WEBUI-22: To make sure the overlay backdrop fills the entire screen in mobile */
+          iron-overlay-backdrop {
```

Comment:
We don't even make use of this element on nuxeo-data-table, right? This is just for the particular case of the iron-overlay-backdrop being inserted by the dialog when reparenting happens right?

Reply:
Exactly, and particularly on mobile devices.